src/SchemaRegistry.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.21;
+
+import { Schema } from "./struct/Schema.sol";
+
+contract SchemaRegistry {
```

Comment:
SchemaRegistry must be upgradeable

Reply:
After some external discussion, we'll push the upgradability in a dedicated ticket later on.